### PR TITLE
Menu and static pages

### PR DESCRIPTION
**Pull request by njason**

### CODE REVIEW

File: src/public/_static-template.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset='utf-8'>
+	<meta name='viewport' content='width=device-width,initial-scale=1'>
+
+	<title>About</title>
+
+	<link rel='icon' type='image/png' href='/favicon.png'>
```
Comment (tzinckgraf) [inline src/public/_static-template.html]: Can we use spaces instead of tabs?

File: src/public/_static-template.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+	<title>About</title>
+
+	<link rel='icon' type='image/png' href='/favicon.png'>
+  <link rel='stylesheet' href='/global.css'>
+	<link rel='stylesheet' href='/static.css'>
+</head>
+
+<body>
+  <nav><div class="inner"><div id="mobile-icon" class="mobile-icon" onclick="toggleMobileMenu()"><div class="middle-line"></div></div> <ul id="navbar-list" class="navbar-list"><li><a href="/">2021 NYC HeatWatch Results</a> </li><li><a href="about.html">About</a> </li><li><a href="resources.html">Resources</a> </li><li><a href="media.html">Media Coverage</a> </li></ul></div></nav>
```
Comment (tzinckgraf) [inline src/public/_static-template.html:15]: Can this be put into multiple lines?

Comment (njason) [reply src/public/_static-template.html:15]: It's just what gets generated from svelte and is redundant on each page. I didn't want to waste the space making it pretty so it's easier to edit the static content

File: src/public/_static-template.html (hunk 1 above)
Comment (njason) [reply src/public/_static-template.html]: Fixed

**Review by tzinckgraf — APPROVED** [review]

**Pull request merged**